doc/dev/crimson/crimson.rst

```diff
@@ -146,6 +124,106 @@ Running Crimson
 
 As you might expect, Crimson does not yet have as extensive a feature set as does ``ceph-osd``.
 
+vstart.sh
+=========
+
+The following options can be used with ``vstart.sh``.
+
+``--crimson``
+    Start ``crimson-osd`` instead of ``ceph-osd``.
+
+``--nodaemon``
+    Do not daemonize the service.
+
+``--redirect-output``
+    Redirect the ``stdout`` and ``stderr`` to ``out/$type.$num.stdout``.
+
+``--osd-args``
+    Pass extra command line options to ``crimson-osd`` or ``ceph-osd``.
+    This is useful for passing Seastar options to ``crimson-osd``. For
+    example, one can supply ``--osd-args "--memory 2G"`` to set the amount of
+    memory to use. Please refer to the output of::
+
+      crimson-osd --help-seastar
+
+    for additional Seastar-specific command line options.
+
+``--crimson-smp``
+    The number of cores to use for each OSD.
+    If BlueStore is used, the balance of available cores
+    (as determined by `nproc`) will be assigned to the object store.
+
+``--bluestore``
+    Use the alienized BlueStore as the object store backend. This is the default (see below section on the `object store backend`_ for more details)
```

Comment:
back end

Reply:
Thanks for the review btw 🙂

Reply:
@gitkenan,
It depends on whether it's a noun (and if it is a noun, then it's "back end") or it's an adjective (and if it's an adjective, then it's "backend"). I know that this is extremely finicky, but that's English. Don't worry too much about it. We'll correct any nounal or adjectival errors later. I appreciate your work on this. It's exactly the kind of thing that I want to encourage.

I should introduce myself. I'm Zac, the upstream Ceph docs guy. You're welcome to write to me on Slack any time for help with docs-related things. I should be awake today through most of the Manchester-time-zone workday.

Reply:
Thanks very much! I appreciate that a lot, since it'll require careful sifting through the document, I'll leave that change out of this one as you've suggested. 

Great to meet you! I'm Kenan - I'm a new Ceph developer working on introducing some new plugins, but I'm particularly interested in Crimson at the moment in my spare time (I am working at IBM but I'm doing extra stuff outside of work)

Reply:
btw I'm also based in Manchester UK! 😄

Reply:
I defer to Zac